spec/controllers/graphql/queries/freaks_spec.rb

```diff
@@ -12,6 +12,22 @@ module Graphql
       }
     end
 
-    it { is_expected.to match_response_for(query: :Freaks, sample: :freaks) }
+    before do
+      create(:freak, name: 'Ion')
+      create(:freak, name: 'Gheorghe')
+    end
+
+    it { is_expected.to match_response_for(query: :freaks, sample: :freaks) }
+
+    context 'with pagination params' do
```

Comment:
Why are there pagination related specs here?

Reply:
we unified the spec for the freaks node